.github/workflows/release.yml

```diff
@@ -141,7 +141,7 @@ jobs:
         shell: bash
         run: |
           [ -f ~/.cargo/env ] && source ~/.cargo/env ; cargo build --release && cargo build --profile=dev --bin sui
-          cd external-crates/move && cargo build -p move-analyzer --release
+          cargo build -p sui-move-lsp --release
```

Comment:
@awelc Do we need to call this out specifically? Does it build as part of `cargo build --release`?

Reply:
It is indeed!